test/binary/binary-expire.test.py

```diff
@@ -41,15 +41,15 @@ def check(key, flags, val, level = 0):
 
 stat = mc.stat("reset")
 
-for i in xrange(10000):
+for i in range(10000):
     mc.set('key-%d' % i, 'value-%d' % i, expire=1)
 
 stat = mc.stat()
-while int(stat.get('evictions', '0')) < 10000:
+while int(stat.get(b'evictions', '0')) < 10000:
```

Comment:
It seems convenient to convert keys into strings (`str` type disregarding Python version) right in `mc.stat()`. At least python-binary-memcached does it: see `protocol.py`.

Feel free to resolve without changes, this is cosmetic.

Reply:
Updated, thanks!

```diff
diff --git a/test/binary/binary-expire.test.py b/test/binary/binary-expire.test.py
index f8f293f..50c2698 100644
--- a/test/binary/binary-expire.test.py
+++ b/test/binary/binary-expire.test.py
@@ -45,11 +45,11 @@ for i in range(10000):
     mc.set('key-%d' % i, 'value-%d' % i, expire=1)
 
 stat = mc.stat()
-while int(stat.get(b'evictions', '0')) < 10000:
+while int(stat.get('evictions', '0')) < 10000:
     time.sleep(0.01)
     stat = mc.stat()
 
-issert(b'evictions' in stat)
-iequal(int(mc.stat().get(b'evictions', 0)), 10000)
+issert('evictions' in stat)
+iequal(int(mc.stat().get('evictions', 0)), 10000)
 
 sys.path = saved_path
diff --git a/test/binary/binary.test.py b/test/binary/binary.test.py
index 319eb1b..51a77fc 100644
--- a/test/binary/binary.test.py
+++ b/test/binary/binary.test.py
@@ -251,7 +251,7 @@ empty("x")
 empty("y")
 
 stat2 = mc.stat()
-iequal(int(stat1[b'cmd_flush']) + 1, int(stat2[b'cmd_flush']))
+iequal(int(stat1['cmd_flush']) + 1, int(stat2['cmd_flush']))
 
 print("""#----------------------------# diagnostics append #---------------------------#""")
 key, value, suffix = "appendkey", "some value", " more"
diff --git a/test/internal/memcached_connection.py b/test/internal/memcached_connection.py
index 2b742fa..a707f1b 100644
--- a/test/internal/memcached_connection.py
+++ b/test/internal/memcached_connection.py
@@ -481,7 +485,7 @@ class MemcachedBinaryConnection(TarantoolConnection):
             if 'key' in rv and not rv['key'] and \
                'val' in rv and not rv['val']:
                 return ans
-            ans[rv['key']] = rv['val']
+            ans[bytes_to_str(rv['key'])] = rv['val']
         return ans
 
     @binary_decorator
```